client/src/pages/Calendar.js

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from "react";
+import Alert from "react-bootstrap/Alert";
+
+const Calendar = () => {
+	const [bookingsData, setBookingsData] = useState([]);
+	const [error, setError] = useState(null);
+	const [isLoading, setIsLoading] = useState(false);
+
+	useEffect(() => {
+		setIsLoading(true);
+		fetch("/api/bookings?filter=2weeks")
+			.then((response) => {
+				if (!response.ok) {
+					throw Error(
+						"There was an error getting the calendar data, Please try again"
+					);
+				}
+				return response.json();
+			})
+			.then((data) => {
+				setBookingsData(data);
+				setError(null);
```

Comment:
```suggestion
```
We don't need to set error to null as it is already null

Reply:
Ok, already changed.